### PR TITLE
[Stream] AV1 beta docs, changelog entry

### PR DESCRIPTION
**Pull request by irvinebroque**
- Adds docs on how to try AV1 with your live stream or recording
- Adds a changelog entry

### CODE REVIEW

**Review by dcpena — APPROVED** [review]

File: content/stream/viewing-videos/av1-playback.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+header: Example DASH manifest URL, with betaCodecSuggestion=av1 query parameter
+highlight: [1]
+https://customer-wi9sckcs7uxt7lh4.cloudflarestream.com/bb2eaea47dd47e87ecde343451b3e8a1/manifest/video.mpd?betaCodecSuggestion=av1
+```
+
+You can find the HLS and DASH manifest URLs for a live stream using the [dashboard](/stream/stream-live/watch-live-stream/#use-the-dashboard) or the [Stream API](/stream/stream-live/watch-live-stream/#use-the-api). For more on how to use HLS and DASH URLs with your video player, read [the docs](/stream/viewing-videos/using-own-player/).
+
+Note that while in beta, only live streams or live recordings created after October 1st can be played using AV1.
```
Comment (Everlag) [inline content/stream/viewing-videos/av1-playback.md:32]: Hard to phrase but is there a way to get across 'your live input can be older than October 1st but you'll need to restart your broadcast after October 1'

File: content/stream/viewing-videos/av1-playback.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      allowFullScreen></iframe>
+  </div>
+</figure>
+
+## Browser and device support for AV1
+
+- Chrome (Desktop and Android) supports AV1 playback
+- Firefox (Desktop) supports AV1 playback
+- Safari does not support AV1 playback
+- Edge only supports AV1 playback if the [AV1 Extention](https://www.microsoft.com/store/productId/9MVZQVXJBQ9V) is installed
```
Comment (Everlag) [inline content/stream/viewing-videos/av1-playback.md]: s/Extention/Extension/?

File: content/stream/viewing-videos/av1-playback.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+      allowFullScreen></iframe>
+  </div>
+</figure>
+
+## Browser and device support for AV1
+
+- Chrome (Desktop and Android) supports AV1 playback
+- Firefox (Desktop) supports AV1 playback
+- Safari does not support AV1 playback
```
Comment (Everlag) [inline content/stream/viewing-videos/av1-playback.md]: Maybe consider `does not currently support` to reduce the finality of the statement?

**Review by Everlag — APPROVED** [review]
Mostly LGTM apart from comments

File: content/stream/viewing-videos/av1-playback.md (hunk 2 above)
Comment (irvinebroque) [inline content/stream/viewing-videos/av1-playback.md]:
```suggestion
- Edge only supports AV1 playback if the [AV1 Extension](https://www.microsoft.com/store/productId/9MVZQVXJBQ9V) is installed
```

File: content/stream/viewing-videos/av1-playback.md (hunk 3 above)
Comment (irvinebroque) [inline content/stream/viewing-videos/av1-playback.md]:
```suggestion
- Safari does not currently support AV1 playback
```

**Review by Schachte — APPROVED** [review]
lg2m!

**Pull request merged**